cli/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/Masterminds/sprig/v3 v3.1.0 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef
```

Comment:
I'm not sure what's the best way here to check if the input is a URL or the path to a file